packages/esm-patient-orders-app/src/order-cancellation-form/cancel-order-form.component.tsx

```diff
@@ -46,8 +46,8 @@ const OrderCancellationForm: React.FC<OrderCancellationFormProps> = ({
         .refine((date) => date >= dayjs().startOf('day').toDate(), {
           message: t('dateCannotBeBeforeToday', 'Date cannot be before today'),
         }),
-      reasonForCancellation: z.string({
-        required_error: t('reasonForCancellationRequired', 'Reason for cancellation is required'),
```

Comment:
The original check only validates that the field input is not null or undefined. The `min(1)` check explicitly validates that the string is not empty. It's also arguably more readable.